app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
@a17r Who maintains the libreoffice _ebuilds_?  I fixed the other _ebuilds_ for you, but this is as much time as I have to spend on this problem.  At least two people are still waiting for this to be fixed.